articles/openshift/howto-gpu-workloads.md

```diff
@@ -188,6 +188,36 @@ ARO uses Kubernetes MachineSet to create machine sets. The procedure below expla
 
 1. Verify the other data in the yaml file.
 
+#### Ensure the correct SKU is set
+
+Depending on the image used for the machine set, both values for `image.sku` and `image.version` must be set accordingly. This is to ensure if generation 1 or 2 virtual machine for Hyper-V will be used. More details [here](https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/plan/should-i-create-a-generation-1-or-2-virtual-machine-in-hyper-v).
```

Comment:
Please change the format of the link to say "See xxx for more information."

Reply:
I am not completely sure if my new commit is the way that you intended. Please let me know if need further adjustments.